source/cloud-native-platform/common-pipeline/publishing-libraries/java.html.md.erb

```diff
@@ -8,9 +8,71 @@ hide_in_navigation: true
 
 # <%= current_page.data.title %>
 
-For publishing Java libraries, we use JitPack as our artifact repository and GitHub Actions to publish the libraries.
+For publishing Java libraries, we now use Azure Artifacts (preferred platform default). Previously, we used JitPack as our artifact repository to publish the libraries, which is now deprecated and scheduled for removal from the platform.
 
-## Publishing with JitPack
+
+## Publishing with Azure DevOps Artifacts
+You will need to set up a Github Action in your repo, using a template as below as a starting point.
```

Comment:
doesn't read correct to me (maybe just me :D ) - where is the GitHub action references? all I can see is ADO

Reply:
rtfm